hyko_toolkit/apis/cohere/chat/metadata.py

```diff
@@ -0,0 +1,87 @@
+from enum import Enum
+from typing import List
+
+import httpx
+from hyko_sdk.models import CoreModel, Method
+from pydantic import Field
+
+from hyko_toolkit.apis.api_registry import ToolkitAPI
+from hyko_toolkit.exceptions import APICallError
+
+func = ToolkitAPI(
+    name="cohere_chat_api",
+    task="cohere",
+    description="Use cohere api for text generation.",
+)
+
+
+class Model(str, Enum):
```

Comment:
enums with one value will break the frontend, either support other values or make this a fixed value